src/clr-icons/api-autoload.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClarityIconsApi } from './api';
```

Comment:
Let's rename this to `api-autoload.umd.ts`for consistency with naming umd files.